apps/chat/src/components/Folder/Folder.tsx

```diff
@@ -685,7 +685,7 @@ const Folder = <T extends ConversationInfo | PromptInfo | DialFile>({
     >
       <div
         className={classNames(
-          'group/folder-item group relative flex h-[30px] items-center rounded border-l-2 hover:bg-accent-primary-alpha',
+          'group/button group/folder-item group relative flex h-[30px] items-center rounded border-l-2 hover:bg-accent-primary-alpha',
```

Comment:
Why do you need `group/button` class here?

Reply:
Previously, the hover was associated with the block below and this prevented the tooltip from being implemented, because during the hover the tooltip appeared in the middle of the entire line and not in the middle of the text, and it looked wrong. Therefore, I raised the hover to a higher level so that the tooltip appears only on the title hover, and during the hover, a button on the right for settings appears on the entire line.